cmd/aro/dbtoken.go

```diff
@@ -83,7 +83,7 @@ func dbtoken(ctx context.Context, log *logrus.Entry) error {
 		return err
 	}
 
-	verifier, err := oidc.NewVerifier(ctx, "https://sts.windows.net/"+_env.TenantID()+"/", pkgdbtoken.Resource)
+	verifier, err := oidc.NewVerifier(ctx, _env.Environment().ActiveDirectoryEndpoint+_env.TenantID()+"/", "https://dbtoken."+_env.AROEnvironment().AppSuffix+"/")
```

Comment:
It makes it hard to read now and understand. Can we initiate this above in the string and maybe use fmt.Sprintf? Just for readabilty. And Add a comment what is expected value as example

Reply:
I'm setting variables and adding comments, but I don't think `Sprintf` would be that much clearer. (First one would be `%s%s/`, which is unhelpful. Second one would be `https://dbtoken.%s/`, which is ok I guess)